content/pages/homepage.html

```diff
@@ -23,28 +23,32 @@
           <div class="ui relaxed grid centered middle aligned">
             <div class="row">
 
-              <div class="seven wide computer twelve wide tablet sixteen wide mobile column">
+                <div class="seven wide computer twelve wide tablet sixteen wide mobile column">
                 {% block homepage_callout %}
                   <div class="ui large left aligned basic vertical segment">
-                    <h1 class="ui huge header">
-                      Documentation simplified
-                    </h1>
-                    <p>
-                      Build, host, and share documentation, all with a single platform.
-                    </p>
-
-                    {% block homepage_callout_buttons %}
-                      <a class="ui large teal stackable button"
-                        data-analytics="signup-modal"
-                         onclick="jQuery('#signup-modal').modal('show');">
-                        <i class="fad fa-play icon"></i>
-                        Sign up now
-                      </a>
-                    {% endblock homepage_callout_buttons %}
+                  <h1 class="ui huge header">
+                    Docs as Code for Everyone
+                  </h1>
+                  <p>
+                    Empower your team with versioned documentation, seamless previews, and powerful authentication. All integrated into your Git workflow.
+                  </p>
+
+                  {% block homepage_callout_buttons %}
+                    <a class="ui large teal stackable button"
+                    data-analytics="signup-modal"
+                     onclick="jQuery('#signup-modal').modal('show');">
+                    <i class="fad fa-play icon"></i>
+                    Get Started for Free
+                    </a>
+                    <a href="/features/" class="ui large secondary stackable button">
```

Comment:
It might be worth using a different variant, the buttons look equally important in this CTA. Basic button provides a less prominent looking button:

![image](https://github.com/user-attachments/assets/ccb248ca-da52-4686-a6b3-12ae95f67ea2)

```suggestion
                    <a href="/features/" class="ui large secondary stackable basic button">
```